backend/tests/services/channels/test_private_im_session_integration.py

```
@@ -432,6 +432,199 @@ async def fake_create_streaming_emitter(message_context: MessageContext):
+    monkeypatch.setattr(
+        runtime_work_service,
+        "send_runtime_message",
+        fake_send_runtime_message,
+    )
+
+    handled = await handler.handle_message(_message("继续 runtime"))
+
+    assert handled is True
+    assert "callback" not in calls
+    assert "send" not in calls
+    assert handler.replies == ["当前本地任务不可用，请回到 Wework 重新选择。"]
```

Ruff (0.15.17)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)